EIPS/eip-2502.md

```diff
@@ -0,0 +1,462 @@
+
+The Root topic should be used as parent when a topic represents the root topic of a democracy.
+
+The example below demonstrates the relationship between topics.
+
+```plantuml
+@startuml
+
+object Root
+
+package "SNT Democracy" {
+    object StatusNetwork
+    object StickerMarket
+    object ENSUsernames
+    object TellerNetwork
+    object RemovalRequest
+    object CurationRequest
+    object EditRequest
+    StatusNetwork : defaultDelegate = "StatusMultisig"
+    ENSUsernames : defaultDelegate = "DevMultisig"
+    StatusNetwork <|-- StickerMarket
+    StatusNetwork <|-- ENSUsernames
+    StatusNetwork <|-- TellerNetwork
+
+    StickerMarket <|-- RemovalRequest
+    StickerMarket <|-- EditRequest
+    StickerMarket <|-- CurationRequest
+}
+
+package "ANT Democracy" {
+    object AragonNetwork
+}
+
+package "GNO Democracy" {
+    object GnosisNetwork
+}
+Root <|-- StatusNetwork
+Root <|-- AragonNetwork
+Root <|-- GnosisNetwork
+
+@enduml
+```
+
+#### Root Topic
+
+The root topic is used for defining a one address to all delegations. This is very useful for managing of cold storage wallets, multisigs and other scenarios where regular voting becomes complicated. This hot wallet can then delegate to several other delegates in specific topics, or handle the voting by themselves, as desired by user.
+
+```plantuml
+@startuml
+rectangle Root {
+    :ColdWallet: -> :HotWallet: : delegate
+}
+
+rectangle Topic1 {
+    :HotWallet: -> :Specialist1: : delegate
+}
+
+rectangle Topic2 {
+    :HotWallet: -> :Specialist2: : delegate
+}
+@enduml
+```
+
+#### Sub Delegations (Micro Topics)
+
+Sub Delegations operate under their own rules, as having a Default Delegate, however would be common for them following the same rules as root delegation. 
+
+When an address don't have a delegate set, it looks up in the parent delegation. If the parent delegation returns zero, the default delegate is looked up at the sub delegation. The address zero delegate (default delegate) is only looked up in the sub delegation.
+
+##### Default Delegate
+
+When a user didn't set a delegate in a parent delegation, it will lookup to the defaultDelegate, which in root topic is "undefined", but in sub topics a default delegate can be defined.  
+
+The default delegate is a option sub delegations can implement to facilitate developers of building a DAO, where all stakeholders by default would delegate to the developer address. Some democracies may allow change of default delegate, others not. 
+
+Technically, Default Delegate is the delegate of the address zero, which appears when a delegate was not set. At root delegation, the technical end of linear delegations is the address zero, which ends up in a circular delegation with itself. 
+
+
+### Delegation Interface
+
+```solidity
+pragma solidity >=0.5.0 <0.7.0;
+
+/**
+ * @author Ricardo Guilherme Schmidt (Status Research & Development GmbH)
+ */
```

Comment:
Since everything in the EIPs repository is CC0, attribution is not required and just adds noise to the interface.  Recommend removing.

Also, we are trying to minimize people using EIPs as a form of advertising their product/company, and one strategy to assist with that is strongly discouraging/disallowing people linking externally or referencing external products/businesses.